can_tools/scrapers/official/NC/nc_vaccine.py

```diff
@@ -1,8 +1,10 @@
 import pandas as pd
+from pandas.core.frame import DataFrame
 import us
-
+import datetime as dt
 from can_tools.scrapers.base import CMU
 from can_tools.scrapers.official.base import TableauDashboard
+from tableauscraper import TableauScraper as TS
```

Comment:
The `TableauDashboard` superclass does have methods for extracting data from a tableau dashboard. The other scrapers in the repo are using those tools. I believe we want to utilize that instead of the `tableauscraper` package. @sglyon, can you confirm?

Reply:
So i attempted to leverage those initially, but it didn't have the capability to do subsheet navigation, I found a similar PR leveraging this package so did so here as well

Reply:
Bump here, if there's a way to do subsheet navigation with those tools I will use them, but when we looked previous there was not

Reply:
Just checked latest main, package is already in Main under TableauScraper==0.1.2, so someone else needed it too, removed all diff on adding new package since it's already in amin